### ISSUE TITLE
JP20

### ISSUE THREAD

**Issue by mattgarrish** [daisy/epub-accessibility-iso#20]
“Distribution" does not define requirements on EPUB publications.

Proposed Change: We do not believe that requirements on distribution channels can be well-defined in this document. We propose to drop 6.4.

(Clause 6.4)

**Comment by mattgarrish** [comment]
This section is problematic in terms of enforcement. I don't agree with dropping it, but it could be made informative guidance.

**Comment by mattgarrish** [comment]
Accepted

Action: Same as JP 2, JP 3.

**Comment by mattgarrish** [comment]
I'm going to drop Discovery from the conformance classes and put the requirement to distribute per clause 10 as bullets under the Accessible and Optimized classes.

If Discoverable content doesn't have to be accessible, it also shouldn't have to follow the distribution requirements, so I'm not adding it there.

/cc @GeorgeKerscher @avneeshsingh @murata2makoto

**Comment by murata2makoto** [comment]
Dropping "discovery" (as a conformance class) is a major change!

I think that the current wording allows any combinations such as:

- non-accessible, optimized, and discoverable
- non-accessible, optimized, and non-discoverable

I am not sure if it is a good idea to do this major change at this stage of the game.

**Comment by mattgarrish** [comment]
Oops, sorry, I didn't mean Discoverable I meant Distribution.

The distribution part is supposed to apply to all types of publication, so it's not really a unique "class". We defined three classes of epub publication and then "distribution" was supposed to apply to all of them.

But to avoid inconsistencies, it might be simplest to move the distribution bullet to the Discoverable class. Accessible and Optimized Publications both inherit the requirements of Discoverable, so nothing changes as far as requirements but we get rid of the confusion that "Discoverable" is a unique class of publication.

**Comment by murata2makoto** [comment]
> we get rid of the confusion that "Discoverable" is a unique class of publication.

Distribution?

> But to avoid inconsistencies, it might be simplest to move the distribution bullet to the Discoverable class. 

Are you talking about the following sentence?  It has to be dropped.

> it shall be distributed accessibly, as defined in 10. Distribution."

**Comment by mattgarrish** [comment]
> Distribution?

Yes. I have some kind of mental block when it comes to typing the two.


> Are you talking about the following sentence? It has to be dropped.

Yes, but see my comment in #2. Is there consensus that we are going to publish two different versions of the same specification?

**Comment by mattgarrish** [comment]
And to be clear, if there is consensus, that's fine. My concern is that this is the kind of change that will lead others to object.
